recipe/meta.yaml

```diff
@@ -11,9 +11,10 @@ source:
   patches:
```

Comment:
`source/url` isn't secure `ftp`. It must be `https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz`, see https://ftp.gnu.org/gnu/gdbm/
Can we use `sha256` instead of `sha1`?

Reply:
hmm, not sure.  CF uses sha1, we support sha1, why we need to change this?

Reply:
out of curiosity, what is secure on https over ftp?

Reply:
In my opinion, it's for the uniformity of our recipes. And the old standard SHA1 is less secure than SHA256.